bin/page_traffic_load

```
@@ -23,10 +23,11 @@ Slop.parse(help: true) do
   DOC
 
   run do |_opts, args|
-    if args.size == 1
+    if args.empty?
       GovukIndex::PageTrafficLoader.new.load_from($stdin)
     else
       puts self
+      exit 1
```

What does `puts self` print in this context? Is it helpful, or would it be better to raise an error with a message explaining what went wrong?

---

it prints the `slop` banner described above. 

I'm happy for this functionality to change but would do that as a separate task to this which is just getting it to run.

---

I just checked the Jenkins output to see how this looked in practice, but I don't see the Slop banner here: https://deploy.publishing.service.gov.uk/job/search-fetch-analytics-data/815/console

Am I looking in the right place?

---

That was due to the script being commented out on that date (this was how this job was initially disabled) If you go back to before that change you can see it in the older jobs.